### ISSUE TITLE
terraform plan incorrectly produces DBParameterGroup not found

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#490]
_This issue was originally opened by @syusuf1234 as hashicorp/terraform#11441. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

After successfully creating a RDS instance and a DB parameter group, a `terraform plan` produces error saying the DB parameter group didn't exist while it actually does!

### Terraform Version
0.7.11

### Affected Resource(s)

```terraform state list
aws_db_instance.aws_postgres_instance
module.postgresql.aws_db_parameter_group.dbpg
```

### Expected Behavior
Terraform plan should have detected no changes to be made.

### Actual Behavior
```
Error refreshing state: 1 error(s) occurred:
* aws_db_parameter_group.dbpg: DBParameterGroupNotFound: DBParameterGroup not found: ctda-globalid-pg96-mse
	status code: 404, request id: d0cb8f48-e3f6-11e6-b27f-e722972ee738
```

### Steps to Reproduce

1. Use a module to launch `RDS instance` and an accompanying `parameter group` resources
2. `terraform apply`
3. `terraform state list` to see the resources
4. `terraform state mv aws_db_instance.aws_postgres_instance aws_db_instance.aws_postgres_instance1`
5. `terraform plan`

More details:
terraform show actually lists these resources as being there: 

```
terraform show
aws_db_instance.aws_postgres_instance:
  id = globalid-pgsql-mse-96
  address = globalid-pgsql-mse-96.cmfehoos8pzo.us-west-2.rds.amazonaws.com
  allocated_storage = 100
  <truncated rest of the output as not applicable>

module.postgresql.aws_db_parameter_group.dbpg:
  id = ctda-globalid-pg96-mse
  arn = arn:aws:rds:us-west-2:971383676178:pg:ctda-globalid-pg96-mse
  description = RDS Postgres db parameter group
  family = postgres9.6
  name = ctda-globalid-pg96-mse
  parameter.# = 0
  tags.% = 0
```

**Comment by Ninir** [comment]
Hey @syusuf1234

Following your discussion in https://github.com/hashicorp/terraform/issues/11441, first: do you still have the issue?
If so, could you provide your terraform.tfvars for this?

Thanks!

**Comment by DTTerastar** [comment]
Just happened to me.  It occurred after manually deleting a db param group.

**Comment by korfuri** [comment]
I took a stab at this, I believe #2620 is the solution.

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.7.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#170-january-12-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
